Sources/App/Views/PackageController/PackageShow+Model.swift

```diff
@@ -287,7 +287,9 @@ extension PackageShow.Model {
         }
 
         if authors.numberOfContributors > 0 {
-            nodes.append(.text("\(authors.numberOfContributors) other " + "contributor".pluralized(for: authors.numberOfContributors)))
+            guard let numberOfContributors = NumberFormatter.spiDefault.string(from: authors.numberOfContributors)
+            else { return .empty }
```

Comment:
Just a minor suggestion, we could use an `if let` here instead of a `guard let` and prevent the possibility of this ever killing  the whole section. It's very unlikely to ever fail, but might be worth doing?

```suggestion
"\(authors.numberOfContributors)"
```